framework/components/ecl-navigation/ecl-navigation-menus/_horizontal-menus.scss

```diff
@@ -11,8 +15,7 @@
   flex-grow: 1;
   list-style: none;
   margin: 0;
-  padding-left: 0;
-  transition: 0.2s;
```

Comment:
On the preview https://deploy-preview-363--europa-component-library.netlify.com/components/preview/ecl-navigation-menus#home the animation of the main menu doesn't work any more - is it intentional?